### DISCUSSION TITLE
June 26, 2023 - Stand-Up

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#384] [Q&A] [answered]
@hollowaykeanho , some good news?

**Comment by JeanShuralyov** [comment]
*knock knock* =)

**Comment by hollowaykeanho** [comment, marked as answer]
`v1.2.0` launched on June 26. Epic upgrades and I'm already loving it. Currently running a field testing with my own portfolio. Good news so far.

I might roll out a slight `v1.2.1` minor version for value changes and some small bug fixes (e.g. table column header). Also, some complaint that the light & dark grey for typography is highly annoying on hi-res screen like Apple Retina display.

**Comment by JeanShuralyov** [comment]
Haha... rich people. >< It's good you have such feedback to work on.

**Comment by corygalyna** [comment]
Should I factor in the schedule?

**Comment by hollowaykeanho** [comment]
> Should I factor in the schedule?

No. It's minor and it's not that crazy anyway.

**Comment by corygalyna** [comment]
ok. Anything else? Helps, etc?

**Comment by hollowaykeanho** [comment]
Nope. Make sure you and Jean are safe over there. That will do. =)

**Comment by JeanShuralyov** [comment]
Aww... so sweet.

**Comment by hollowaykeanho** [comment]
If it's nothing else, I want to get back to the field testing. @JeanShuralyov , cya.

@corygalyna, you're free to work on your own pursue now. Don't worry.

**Comment by corygalyna** [comment]
no problem. My contributions are my own. It's sad we have to move into this situation but hey, like you, I'm okay to contribute in once in a while. =)
